extensions/spark/kyuubi-spark-lineage/src/main/scala/org/apache/kyuubi/plugin/lineage/helper/SparkSQLLineageParseHelper.scala

```diff
@@ -70,15 +77,40 @@ trait LineageParser {
       val childMap = child.map { case (k, attrs) => (k.exprId, attrs) }
       parent.map { case (k, attrs) =>
         k -> AttributeSet(attrs.flatMap(attr =>
-          childMap.getOrElse(attr.exprId, AttributeSet.empty)))
+          childMap.getOrElse(
+            attr.exprId,
+            if (attr.name.equalsIgnoreCase(AGGREGATE_COLUMN_IDENTIFIER)) AttributeSet(attr)
+            else AttributeSet.empty)))
       }
     }
   }
 
+  private def getExpressionSubqueryPlans(expression: Expression): Seq[LogicalPlan] = {
+    expression match {
+      case s: ScalarSubquery => Seq(s.plan)
+      case s => s.children.flatMap(getExpressionSubqueryPlans)
+    }
+  }
+
   private def getSelectColumnLineage(
       named: Seq[NamedExpression]): AttributeMap[AttributeSet] = {
     val exps = named.map {
-      case a: Alias => a.toAttribute -> a.references
+      case exp: Alias =>
+        if (exp.references.nonEmpty) exp.toAttribute -> exp.references
+        else {
+          exp.child match {
+            case _: AggregateExpression =>
```

Comment:
is there other cases exclude `count(*)` ?

Reply:
OK, i will add it later. 
Aggregate expressions can generally extract the columns except for count(*) by calling `exp.references`, e.g., `first`, `avg`.

Reply:
`first`, `avg` should be already conveted by the above code, right ?
`if (exp.references.nonEmpty) exp.toAttribute -> exp.references`

I mean if the `count(*)` is the only case we need to handle, we can make it more clear just match a `count(*)` aggregate function and change `__aggregate__ ` to `__count__` ?

Reply:
Yes, I considered you bringing up this situation, but found
```
select every(count(*) == 1) as a from t0
```
`count(*)`  also be nested. So keep `__aggreate__` not changed to `__count__`

Reply:
even it is nested, I think the `__count__ ` should be more accurate ? for example, the input lineage of the query`select count(*) + sum(c1) as c2 from t` will be: `t.__count__` and `t.c1`.

Reply:
Thanks, i can get it now